l2geth/internal/ethapi/api.go

```diff
@@ -1647,18 +1648,30 @@ func (s *PublicTransactionPoolAPI) FillTransaction(ctx context.Context, args Sen
 // SendRawTransaction will add the signed transaction to the transaction pool.
 // The sender is responsible for signing the transaction and using the correct nonce.
 func (s *PublicTransactionPoolAPI) SendRawTransaction(ctx context.Context, encodedTx hexutil.Bytes) (common.Hash, error) {
-	if s.b.IsVerifier() {
-		return common.Hash{}, errors.New("Cannot send raw transaction in verifier mode")
+	tx := new(types.Transaction)
+	if err := rlp.DecodeBytes(encodedTx, tx); err != nil {
+		return common.Hash{}, err
 	}
 
 	if s.b.IsSyncing() {
 		return common.Hash{}, errors.New("Cannot send raw transaction while syncing")
 	}
 
-	tx := new(types.Transaction)
-	if err := rlp.DecodeBytes(encodedTx, tx); err != nil {
-		return common.Hash{}, err
+	// Forward tx to sequencer if config is set
+	// TODO: add forwarding config
+	if s.b.IsVerifier() {
+		// TODO: replace with env var L2_URL
```

Comment:
An example for how to parse and pass through the config is here:
- https://github.com/ethereum-optimism/optimism/blob/6749ab3d3c8d07c9d4ca4a8b4eac06a9380eefc4/l2geth/cmd/utils/flags.go#L814
- https://github.com/ethereum-optimism/optimism/blob/6749ab3d3c8d07c9d4ca4a8b4eac06a9380eefc4/l2geth/cmd/utils/flags.go#L1104
- https://github.com/ethereum-optimism/optimism/blob/6749ab3d3c8d07c9d4ca4a8b4eac06a9380eefc4/l2geth/eth/backend.go#L227

Reply:
Thanks, using that now.